code/modules/mob/living/simple_animal/hostile/wastebots.dm

```diff
@@ -30,8 +30,8 @@
 	icon_state = "gutsy"
 	icon_living = "gutsy"
 	icon_dead = "gib7"
-	health = 200
-	maxHealth = 200
+	health = 400
```

Comment:
honk, sync your master branch!

Reply:
Yeah, I fucked up after I made those mob changes and merged early, then there were issues when I reverted, etc. What's the easiest way to fix all that shiiiet?

Reply:
just set the var to 200

Reply:
Ok, fixed those. I'm just going to remake my fork when my pull requests go through. Idk how to unbork those master commits :/